src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -66,18 +66,19 @@
                                                      (:header-darkgray style/colors)}}
                                         children])
                  cell (fn [children] [:span {:style {:paddingLeft 16}} children])]
-             {:columns [{:header-component [:div {:style {:padding "13px 0 12px 12px"
-                                                          :backgroundColor (:header-darkgray style/colors)}}
-                                            [:input {:type "checkbox" :ref "imcallcheck"}]]
-                         :starting-width 42 :resizable? false
+             {:columns [{:header-component (header "Name") :starting-width 200
                          :cell-renderer (fn [row-num conf]
-                                          [:div {:style {:paddingLeft 12}}
-                                           [:input {:type "checkbox" :ref (str "imc_" row-num)}]])}
-                        {:header-component (header "Name") :starting-width 200
-                         :cell-renderer (fn [row-num conf]
-                                          (cell [:a {:href "javascript:;"
-                                                     :style {:color (:button-blue style/colors)
-                                                             :textDecoration "none"}} (conf "name")]))}
+                                          (cell [:a {:onClick
+                                                     (fn [] (js/alert
+                                                              (str "TODO : process/prompt for "
+                                                                "import of this method configuration "
+                                                                conf " into the workspace →"
+                                                                (get props :workspace) )))
+                                                     :href "javascript:;"
+                                                     :style {:color
+                                                              (:button-blue style/colors)
```

Comment:
@dmohs  Regarding this note "Merge with previous line." about line 80, I specifically added the newline in line 79 after "(:button-blue style/colors)" so that the line wouldn't break the
rule "When feasible, avoid making lines longer than 100 characters."  (  from here https://broadinstitute.atlassian.net/wiki/display/DSDE/Code+Style+Conventions ) .  I did so because In my judgement and determination I considered the insertion of the line to be "feasible"